web/components/Home/Hero.js

```diff
@@ -14,7 +14,11 @@ export default function Hero({ title, about }) {
     <Section>
       <div className="pt-30 sm:px-0 px-10 pb-40">
         <div className="top-5 left-5 absolute">
-          <Logo />
+          <Link href="/">
+            <a>
```

Comment:
Why adding "a" tag inside Link, I think Link is enough in React

Reply:
this is how it's done https://nextjs.org/docs/routing/introduction#linking-between-pages